design/resources.go

```diff
@@ -5,6 +5,57 @@ import (
 	. "github.com/goadesign/goa/design/apidsl"
 )
 
+var _ = Resource("workitem", func() {
+	BasePath("/workitem")
+
+	Action("show", func() {
+		Routing(
+			GET("/:id"),
+		)
+		Description("Retrieve work item with given id.")
+		Params(func() {
+			Param("id", String, "id")
+		})
+		Response(OK, func() {
+			Media(WorkItem)
+		})
+		Response(NotFound)
+	})
+	
+	Action("create", func() {
+		Routing(
+			POST(""),
+		)
+		Description("create work item with type and id.")
+		Payload(CreateWorkItemPayload)
+		Response(OK, func() {
```

Comment:
I believe this should return `CREATED` (201) and ideally `href` to this resource as part of the reponse.